releasenotes/notes/upgrade-curve-fit-4dc01b1db55ee398.yaml

```diff
@@ -0,0 +1,46 @@
+---
+upgrade:
+  - |
+    The :class:`CurveAnalysis` class has been updated to use the covariance between fit
+    parameters in the error propagation. This will provide more accurate standard
+    error for your fit values.
+  - |
+    The data format of analysis result data value has been replaced from
+    :class:`FitVal` to :class:`UFloat` to support error propatation for
+    post analysis computation.
+
+    .. code-block:: python
+
+      expdata = T1(0, delays, backend).run()
+      fit_t1 = expdata.analysis_results("T1").value
+
+      assert isinstance(fit_t1, UFloat)
+
+      new_value = fit_t1 / 2
+      new_value.std_dev  # show new standard error value
+
+    Now ``fit_t1`` value is :class:`UFloat` instance with
```

Comment:
I don't think we need to duplicate documentation of their classes in our docs, so these links for classes and methods should be removed and we should just link to their docs. 

They also never explicitly mention the `UFloat` class in their user guide so maybe better to use `ufloat` instead.